api/express.ts

```diff
@@ -53,7 +55,7 @@ app.use('/api', openRouter.healthRouter);
 
 // Routing Protected Routes
 // Allow for removed protection when API testing
-const routeProtector = `${process.env.TESTING}`.toLowerCase() == 'true' ? (request, response, next) => { next(); } : protect;
+const routeProtector: any = `${process.env.TESTING}`.toLowerCase() === 'true' ? (request: Request, response: Response, next: NextFunction) => { next(); } : protect;
```

Comment:
```suggestion
const routeProtector: Function = `${process.env.TESTING}`.toLowerCase() === 'true' ? (request: Request, response: Response, next: NextFunction) => { next(); } : protect;
```

Reply:
It doesn't accept Function in app.use(). It looks like it should accept types of RequestHandler, but no luck with that yet either.